dependencies/required.txt

```diff
@@ -21,3 +21,5 @@ scipy < 1.6 ; python_version < "3.7"
 scipy ; python_version >= "3.7"
 pandas < 1.2 ; python_version < "3.7"
 matplotlib < 3.4 ; python_version < "3.7"
+tornado
```

Comment:
Why do we need tornado as required dep?

Reply:
because when I pip install nni in a new env, these two packages are required.

Reply:
![image](https://user-images.githubusercontent.com/33053116/128653176-e5a7ec3a-0473-420b-aec0-3d60a660ddb0.png)

Reply:
reset the requirement